src/Dropdown/index.tsx

```diff
@@ -0,0 +1,18 @@
+import * as React from "react";
+import { Dropdown as AntDropdown, DropDownProps as AntDropdownProps } from "antd";
+
+import "./style.css"
+
+export type DropdownProps = AntDropdownProps;
+
+const Dropdown: React.FC<DropdownProps> = (props) => {
+  return(
+    <AntDropdown overlay={props.overlay} trigger={['click']}>
+      <a className="ant-dropdown-link" onClick={e => e.preventDefault()}>
```

Comment:
The `<a>` element is part of the story not the component

Reply:
also, because `overlay` and `trigger` is taken from the prop, you can remove them and put `{...props}` instead, and place the `overlay` and `trigger` values in the stories instead.